apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -121,6 +121,33 @@ const styles = {
   }
 };
 
+const collapserIconStyle = {
+  showHideButton: {
+    position: 'absolute',
+    top: 0,
+    margin: 0,
+    lineHeight: styleConstants['workspace-headers-height'] + 'px',
+    fontSize: 18,
+    ':hover': {
+      cursor: 'pointer',
+      color: color.white
+    }
+  },
+  showHideButtonLtr: {
```

Comment:
Is there a case in which these two values would be different?

Reply:
Yeah, there's an `isRtl` prop passed to this component (I have to admit I didn't dig in to when it's true or false), but it will determine whether we use the `showHideButtonLtr` or `showHideButtonRtl` class. Was that what you were asking?